lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -16,10 +17,9 @@ def initialize(serializer, options = {})
         end
 
         def serializable_hash(options = nil)
-          options = {}
```

Comment:
I am quite confused now...hahaha
is the first not the same than the second example below?
```ruby
def serializable_hash(options = nil)
   options ||= {}
```
```ruby
def serializable_hash(options = {})
```

which bug was that?